README.md

```diff
@@ -1,14 +1,332 @@
-near-workspaces
-===============
+<div align="center">
 
-Controlled, concurrent workspaces in local [NEAR Sandbox](https://github.com/near/sandbox) blockchains or on [NEAR TestNet](https://docs.near.org/docs/concepts/networks). Fun, deterministic testing and powerful scripting for NEAR.
+  <h1>NEAR Workspaces (TypeScript/JavaScript Edition)</h1>
 
-This repo is setup as a monorepo. Currently it only has one package: [near-workspaces-js]. See the README for more specific info:.
+  [![Project license](https://img.shields.io/badge/license-Apache2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+  [![Project license](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+  [![Discord](https://img.shields.io/discord/490367152054992913?label=discord)](https://discord.gg/Vyp7ETM)
+  [![NPM version](https://img.shields.io/npm/v/near-workspaces.svg?style=flat-square)](https://npmjs.com/near-workspaces)
+  [![Size on NPM](https://img.shields.io/bundlephobia/minzip/near-workspaces.svg?style=flat-square)](https://npmjs.com/near-workspaces)
 
-  [near-workspaces-js]: ./packages/js
+</div>
 
+`NEAR Workspaces` is a library for automating workflows and writing tests for NEAR smart contracts. You can use it as is or integrate with test runner of your choise. We suggest you to use AVA.
```

Comment:
```suggestion
`NEAR Workspaces` is a library for automating workflows and writing tests for NEAR smart contracts. You can use it as is or integrate with test runner of your choise (AVA, Jest, Mocha, etc.). If you don't have a preference, we suggest you to use AVA.
```

Reply:
Original wording is also good. My suggestion want to emphasize we are having good support on well known test runners. And we do have plan to add examples with jest and mocha